src/main/java/com/gmail/val59000mc/scenarios/Scenario.java

```diff
@@ -54,6 +54,7 @@
     NOGOINGBACK("No Going Back", UniversalMaterial.NETHER_BRICK, null, "&6No Going Back&7:", "&7- Once you go through a nether portal in the overworld, you can't go back to the overworld."),
     DOUBLEDATES("Double Dates", UniversalMaterial.RED_BANNER, DoubleDatesListener.class, "&6Double Dates&7:", "&7- Players join with a chosen team and are combined randomly with another team."),
     FLYHIGH("Fly High", UniversalMaterial.ELYTRA, FlyHighListener.class, 9, "&6Fly High&7:", "&7- Players spawn with a elytra."),
+    RANDOMITEMS("Random Items", UniversalMaterial.EXPERIENCE_BOTTLE, RandomItemsListener.class, "&6Random Drops!&7:", "&7- Every block broken will drop random items." ),
```

Comment:
In the description the scenario is called Random drops but the scenario name is Random Items. I'd change both of them to "Randomized Items" as a scenario I recently added is called "Randomized Crafts" so that would match. (Don't forget to rename the listener class and the enum)
In the description the scenario name also has an explanation mark wich I don't think is needed as it would be the only scenario having one.
And maybe change the description to "drop a random item" as it only drops one.

Reply:
changed!